src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs

```diff
@@ -79,6 +83,7 @@ public void ReadLines_CreateString_ReturnResult()
         [Test]
         public void GetWordsOrderedAlphabetically_SortWords_ReturnSortedList()
         {
+            // This is an Analyzer test not FilePartReader test -> should be in separated test file
             _filePartReader.Setup("test.txt", 2, 6);
```

Comment:
`text.txt` arise multiple places. In this case extract it into a private field or into the test `Setup` method.